### ISSUE TITLE
"Start" button disabled when a process doesn't have a form in the start event of the process.

### ISSUE THREAD

**Issue by cijujoseph** [Alfresco/alfresco-ng2-components#1526]
**Type of issue:**  (check with "[x]")
```
- [ ] New feature request
- [ x] Bug  
- [ ] Support request
```
**Current behavior:**
I built a demo adf app and tried to start a process which doesn't have a start form. However the start button is disabled and I'm unable to start the process.
Please find attached a screenshot.
<img width="1395" alt="screen shot 2017-01-23 at 6 22 37 pm" src="https://cloud.githubusercontent.com/assets/2080360/22228842/5837457c-e199-11e6-86ee-cf8951b84fff.png">

**Expected behavior:**
A process without a start-form is a valid scenario and it should be handled by the ng2-process-list component.

**Steps to reproduce the issue:**
create a process without a start form and deploy it to Activiti. Try to start the process via demo shell adf app.

**Component name and version:** 
ng2-process-list v1.0.0

**Browser and version:** 
Tested in Chrome, possibly in all.

**Comment by mauriziovitale** [comment]
I'm not able to reproduce this bug, I can start a process that doesn't have a form attached to the start event.

**Comment by cijujoseph** [comment]
This seems to be fixed in 1.1

**Comment by magemello** [comment]
closing thnks @cijujoseph  and @mauriziovitale
